src/include/processor/warning_context.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <vector>
+
+#include "common/types/types.h"
+#include "main/client_context.h"
+#include "processor/operator/persistent/reader/csv/csv_error.h"
+#include "processor/result/factorized_table.h"
+
+namespace kuzu {
+namespace processor {
+
+struct WarningSchema {
+    static constexpr uint64_t getNumColumns() { return 4; }
```

Comment:
Define this as a static constexpr in constants.h copy section.

Reply:
updated, let me know if the new changes in `constants.h` look good